src/z3c/relationfield/interfaces.py

```diff
@@ -101,3 +103,6 @@ def convert():
 
         Returns real relation object
         """
+
+class IRelationBrokenEvent(IObjectModifiedEvent):
+    pass
```

Comment:
This needs a docstring.

(Usually it's better to document the interface and then have the implementation's docstring refer to it.)